src/azure-cli/azure/cli/command_modules/batch/_command_type.py

```diff
@@ -4,7 +4,14 @@
 # --------------------------------------------------------------------------------------------
 
 import re
-
+import inspect
+import importlib
+
+from enum import Enum
+from tokenize import String
+from typing import Optional, get_type_hints, get_origin, get_args
+from git import Union
```

Comment:
May I know why this is imported? This caused CI failure:
```
Error loading command module 'batch': No module named 'git'
```
If it's not necessary, can you remove import from `git`?

Reply:
Agreed this looks like a mistake. Removing it.